exotica_python/src/pyexotica/tools.py

```diff
@@ -27,6 +29,29 @@ def check_trajectory_continuous_time(scene, trajectory):
     print("Continuous-time collision verification took", end_time - start_time)
     return all_good
 
+
+def check_whether_trajectory_is_collision_free_by_subsampling(scene, trajectory, num_subsamples=10, debug=False):
+    '''
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First line should end with a period, question mark, or exclamation point (not 's') (D415)](https://app.codacy.com/gh/ipab-slmc/exotica/pullRequest?prid=5476070)